src/app/components/shell/header/header.scss

```diff
@@ -44,182 +73,587 @@
+
+            @at-root .sticky#{&} {
+                position: fixed;
+                top: -8rem;
+                left: 0;
+                width: 100%;
+                transition: all 0.5s;
+                background: color(neutral-lightest);
+            }
+
+            @at-root .visible#{&} {
+                top: 0;
+                transition: top 0.5s 0.3s;
             }
         }
-    }
 
-    .logo-quote {
-        display: none;
-        font-size: 1.4rem;
-        margin-top: 0.3rem;
+        > .container {
+            display: flex;
+            justify-content: space-between;
+            padding: 1.5rem;
+            height: 100%;
 
-        @media (min-width: 107rem) {
-            display: inline-block;
+            @include desktop {
+                align-items: center;
+                position: relative;
+            }
         }
-    }
 
-    .nav-menu {
-        align-items: center;
-        display: flex;
-        flex: 0 0 auto;
-        flex-wrap: wrap;
-        list-style-type: none;
-        margin: 0;
-        padding: 0;
-    }
+        .logo-wrapper {
+            line-height: 0;
+            order: 1;
 
-    .nav-menu-item {
-        > a {
+            > a {
+                display: inline-block;
+                width: 14.5rem;
+                transition: all 0.3s linear;
+            }
+
+            svg .st5 {
+                opacity: 1;
+                transition: opacity 0.5s;
+
+                @media (max-width: #{breakpoints(tablet)})  {
+                    @at-root .active#{&} {
+                        opacity: 0;
+                        transition: opacity 0.5s;
+                    }
+                }
+            }
+
+            svg #grey {
+                @media (max-width: #{breakpoints(tablet)})  {
+                    @at-root .active#{&} {
+                        opacity: 1;
+                    }
+                }
+            }
+
+            .logo-quote {
+                display: inline-block;
+                transform: scale3d(0, 0, 0);
+                color: color(gray);
+                font-size: 1.4rem;
+                position: absolute;
+                vertical-align: text-top;
+                margin-left: 2rem;
+                top: -0.1rem;
+
+                @media (min-width: 1000px)  {
+                    position: relative;
+                    transform: scale3d(1, 1, 1);
+                    transition: transform 0.3s;
+                }
+            }
+        } //end logo-wrapper
+
+        .nav-menu {
+            padding: 0 0 0 8rem;
+            list-style: none;
+            flex: 1 100%;
+            order: 3;
+            margin: 1.5rem 0 0;
+            //transform: translate3d(0, 0, 0);
+
+            @media (max-width: #{breakpoints(tablet)})  {
+                @at-root .active.open#{&} {
+                    transform: translate3d(-1000px, 0, 0);
+                    transition: transform 0.2s;
+                }
+            }
```

Comment:
I don't think this media query is necessary.  As written, it applies to phones and tablets, but not desktop.  However, there's already a rule to override these rules below just for desktops.  Therefore, these should work if just included normally under `.nav-menu` without being wrapped in a media query.